VAMobile/src/screens/HealthScreen/SecureMessaging/ReplyMessage/ReplyMessage.tsx

```diff
@@ -139,12 +140,13 @@ const ReplyMessage: FC<ReplyMessageProps> = ({ navigation, route }) => {
   useEffect(() => {
     if (saveDraftComplete) {
       dispatch(updateSecureMessagingTab(SegmentedControlIndexes.FOLDERS))
-      navigation.navigate('SecureMessaging')
-      navigation.navigate('FolderMessages', {
-        folderID: SecureMessagingSystemFolderIdConstants.DRAFTS,
-        folderName: FolderNameTypeConstants.drafts,
-        draftSaved: true,
-      })
+      waygateNativeAlert('WG_SecureMessaging') && navigation.navigate('SecureMessaging')
```

Comment:
Interesting that this navigates to SecureMessaging and then immediately to FolderMessages. Wonder if it's really necessary to do that. Out of scope for this ticket but just something I noticed

Reply:
i believe this is implemented this way due to how the stack may be different than expected behavior. There likely is a better of different way to do this that doesn't require this